odl/tomo/backends/astra_setup.py

```diff
@@ -356,7 +356,8 @@ def astra_projection_geometry(geometry):
         det_width = geometry.det_partition.cell_sides[0]
         det_count = geometry.detector.size
         # convention in 'astra_conebeam_2d_geom_to_vec' differs from ASTRA's
-        angles = geometry.angles - np.pi / 2
+#        angles = geometry.angles - np.pi / 2
+        angles = geometry.angles
```

Comment:
Change to old state. This has been fixed on master, so you get the fix once you rebase on master. However, don't do that now, we should start fresh on a new branch.